src/prometheus-net.DotNetRuntime.Tests/IntegrationTests/KestrelTests.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using NUnit.Framework;
+using Prometheus.DotNetRuntime.EventListening.Parsers;
+using Prometheus.DotNetRuntime.Metrics.Producers;
+
+namespace Prometheus.DotNetRuntime.Tests.IntegrationTests
+{
+    //[TestFixture]
+    //internal class Given_Contention_Events_Are_Enabled_For_Kestrel_Stats 
```

Comment:
I feel like this should have tests but I'm not sure how to write them here.

Reply:
The way I've approached integration testing is to invoke some behaviour in the runtime (e.g. allocate some memory, content a lock) and make sure I can observe the relevant change in the metrics. 

I'd suggest running a web server and issuing traffic against it, verifying that the expected metrics change.